### PR TITLE
Feature/register validations

### PR DESCRIPTION
**Pull request by Sorin37**

### CODE REVIEW

File: SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs [hunk 1]
```diff
@@ -26,12 +26,31 @@ public bool CanExecute(object parameter)
...
-            Session.GeneralSettings = userString;
-            Application.Current.MainPage = new AppShell();
+            string message = await userService.register(parameter as User);
+
+            if (message == "Ok") {
+                string userString = JsonSerializer.Serialize(parameter as User);
+                Session.GeneralSettings = userString;
+                Application.Current.MainPage = new AppShell();
+            }
+            else if (message.Length > 30)
```
Comment (TinelValentin) [inline SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs:38]: Bad Practice!
There could be an exception thrown that is less than 30 characters

File: SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs [hunk 2]
```diff
@@ -26,12 +26,31 @@ public bool CanExecute(object parameter)
...
+            {
+                await App.Current.MainPage.DisplayAlert(
```
Comment (BrassoiCatalin) [inline SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs:40]: Why write the DisplayAlerts so weirdly on many lines? Consider putting them on the same line.

**Review by BrassoiCatalin — CHANGES_REQUESTED** [review]

**Review by TinelValentin — CHANGES_REQUESTED** [review]

File: SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs (hunk 2 above)
Comment (Sorin37) [reply SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs:40]: i think it's more readable like that!

File: SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs (hunk 1 above)
Comment (Sorin37) [reply SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs:38]: there is only one exception that is long, the others are readable and displayable

Comment (TinelValentin) [reply SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs:38]: Still bad practice, but fine

**Review by TinelValentin — APPROVED** [review]

**Review by BrassoiCatalin — APPROVED** [review]

**Pull request merged**